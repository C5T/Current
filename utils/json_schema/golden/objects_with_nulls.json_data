[ {"x":"a","y":null,"z":null,"p":"q"}, {"x":null,"y":"b","z":null,"p":"q"}, {"x":null,"y":null,"z":"c","p":"q"} ]
