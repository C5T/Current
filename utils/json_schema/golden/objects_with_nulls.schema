// Autogenerated schema inferred from input JSON data.

CURRENT_STRUCT(Schema_Element_Object) {
  CURRENT_FIELD(x, Optional<std::string>);
  CURRENT_FIELD(y, Optional<std::string>);
  CURRENT_FIELD(z, Optional<std::string>);
  CURRENT_FIELD(p, std::string);
};

using Schema = std::vector<Schema_Element_Object>;
