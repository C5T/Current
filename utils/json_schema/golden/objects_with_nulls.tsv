Field	Type	Set	Unset/Null	Values	Details
Schema	Array	1	0
Schema[]	Object	3	0	4 fields	x, y, z, p
Schema[].x	String	1	2	1 distinct value, "a":1
Schema[].y	String	1	2	1 distinct value, "b":1
Schema[].z	String	1	2	1 distinct value, "c":1
Schema[].p	String	3	0	1 distinct value, "q":3
